docs/pages/blog/2020-q2-update.md

```diff
@@ -16,7 +16,7 @@ Here are the most significant improvements since March 2020:
   The last 14 months have been spent focusing on improving the library under the v4.x development branch, while not introducing any breaking changes. During this period we have identified several important areas for improvement. While the absence of breaking changes is a significant time-saver for developers, it also limits the scope of the problems that can be solved and the quality of the solutions. We're excited about what comes next!<br /><br />
   You can find the documentation for the next version at https://mui.com/. The next 6-8 months will see weekly releases as always, following [the roadmap](https://github.com/mui/material-ui/issues/20012) and [milestone](https://github.com/mui/material-ui/milestone/35).
 
-- 📍 The icons package has been updated with changes made by Google, leading to [200+ new icons](https://mui.com/components/material-icons/).
+- 📍 The icons package has been updated with changes made by Google, leading to [200+ new icons](https://v4.mui.com/components/material-icons/).
```

Comment:
Why this one does not use the new URL?
```suggestion
- 📍 The icons package has been updated with changes made by Google, leading to [200+ new icons](https://mui.com/material-ui/material-icons/).
```

Reply:
The blog post is sharing an update on the latest change that we introduced 2 years ago. It feels more relevant to link to the closest version of the docs relative to when the post was published.

Reply:
But happy to take this change in.